### ISSUE TITLE
CIFAR10(0) w/ batch size 128

### ISSUE THREAD

**Issue by SamuelGabriel** [ildoonet/pytorch-randaugment#19]
In you configs you use for both CIFAR 10 and 100 you use larger batch sizes than reported in the paper, but the rest of the hyper-parameters are the same. I am rather memory-limited, since I want to save extra statistics, thus I was wondering if you tried smaller batch sizes? If so, I wonder if it worked? Did you just choose that batch size since you could fit it on the GPU?

Best regards and Thank You,

Sam

**Comment by SamuelGabriel** [comment]
I made a few changes to get to the reported results from the paper for cifar10 and wide resent 28x10. The only hyper-parameter changes I made: LR Warm-up (even though it might actually not be needed) and 300 instead of 200 epochs. My initial run gave a result of 97.37.

Here you can see my commit, which branches off the commit where @ildoonet mentioned cifar10 results first: https://github.com/SamuelGabriel/pytorch-randaugment/commit/b9c6de70987ce4f16454856ac4f413cf908cfd39
